Second_Seminar/app/src/main/java/org/sopt/androidseminar/ui/HomeActivity.kt

```diff
@@ -0,0 +1,94 @@
+package org.sopt.androidseminar.ui
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.androidseminar.R
+import org.sopt.androidseminar.adapters.RepositoryAdapter
+import org.sopt.androidseminar.data.RepositoryInfo
+import org.sopt.androidseminar.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityHomeBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        Log.d(ACTIVITY_NAME, "onCreate")
+
+        setRepoRv()
+
+        moreButtonClickedEvent()
+    }
+
+    private fun setRepoRv() {
+        val repoList = mutableListOf<RepositoryInfo>()
+        repoDataInput(repoList)
+
+        val repoAdapter = RepositoryAdapter(repoList)
+        val repoRecyclerView = binding.rvRepository
+        repoRecyclerView.adapter = repoAdapter
+        repoRecyclerView.setHasFixedSize(false)
+        repoAdapter.notifyDataSetChanged()
+    }
+
+    private fun moreButtonClickedEvent() {
+        binding.btMore.setOnClickListener {
+            val intent = Intent(this, FragmentActivity::class.java)
+            startActivity(intent)
+        }
+    }
+
+    private fun repoDataInput(repoList: MutableList<RepositoryInfo>) {
+        for (i in 1..10) {
+            repoList.add(RepositoryInfo("레포지토리 이름".plus(i), "레포지토리 설명".plus(i), "레포지토리 언어".plus(i)))
+        }
```

Comment:
```suggestion
    (1..10).forEach {  repoList.add(RepositoryInfo("레포지토리 이름".plus(it), "레포지토리 설명".plus(it), "레포지토리 언어".plus(it)) }
```

forEach/map/forEachIndexed 등 다양한 함수형 구문을 활용해서 간결하게 루프를 바꾸면 가독성이 올라갑니당